src/profile-logic/symbol-store-db.js

```diff
@@ -238,10 +236,7 @@ export class SymbolStoreDB {
     const lastUsedDateIndex = store.index('lastUsedDate');
     // Get a cursor that walks all records whose lastUsedDate is less than beforeDate.
     const cursorReq = lastUsedDateIndex.openCursor(
-      (window.IDBKeyRange: IDBKeyRange<SymbolDateKey>).upperBound(
-        beforeDate,
-        true
-      )
+      window.IDBKeyRange.upperBound(beforeDate, true)
```

Comment:
See below as I reworked the typing for IDBKeyRange to make it simpler to use (and IIRC I think the newer Flow forced me to change something anyway)